synapse/handlers/room.py

```diff
@@ -450,7 +450,7 @@ async def clone_existing_room(
         old_room_member_state_events = await self.store.get_events(
             old_room_member_state_ids.values()
         )
-        for k, old_event in old_room_member_state_events.items():
+        for old_event in old_room_member_state_events.values():
```

Comment:
mypy complained here that `k` was changing from a `Tuple[str, str]` (essentially a state key) to `str` (event ID).

Turns out we don't use `k` in this loop, so it is easier to just look at the values.